testing/upload_validation.ts

```diff
@@ -50,6 +50,14 @@ import {isNil} from '../helpers/object_utils';
 import {makeSchema} from '../schema';
 import * as z from 'zod';
 
+/**
+ * The uncompiled column format matchers will be expected to be actual regex objects,
+ * and when we compile the pack / stringify it to json, we will store the .toString()
+ * of those regex objects. This regex is used to hydrate the stringified regex back into
+ * a real RegExp object.
+ */
+export const PACKS_VALID_COLUMN_FORMAT_MATCHER_REGEX = /^\/(.*)\/([a-z]+)?$/;
```

Comment:
I assume this is exported because you plan to import on the `coda` side?

Reply:
Yup, for the runtime protection. Happy to just copy/paste it if that feels cleaner to you.